packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMetadataTable.tsx

```diff
@@ -133,7 +131,10 @@ export function CoinMetadataTable(props: CoinMetadataTableProps) {
         },
     )
 
-    useEffect(() => closeMenu(), [position])
+    useEffect(() => {
+        window.addEventListener('scroll', closeMenu)
```

Comment:
Just mention here: Maybe the listening to scroll event is overkill for this scenario, `<ClickAwayListener />` has the better performance.